tests/test_paste2.py

```diff
@@ -19,85 +20,28 @@
 output_dir = test_dir / "data/output"
 
 
-def pytest_generate_tests(metafunc):
-    if "loss_fun" in metafunc.fixturenames:
-        metafunc.parametrize(
-            "loss_fun, filename",
-            [
-                ("square_loss", "gwloss_partial.csv"),
-                ("kl_loss", "gwloss_partial_kl_loss.csv"),
-            ],
-        )
-    if "dissimilarity" in metafunc.fixturenames:
-        metafunc.parametrize(
-            "dissimilarity, filename",
-            [
-                ("euc", "partial_pairwise_align_euc.csv"),
-                ("gkl", "partial_pairwise_align_gkl.csv"),
-                ("kl", "partial_pairwise_align_kl.csv"),
-                ("selection_kl", "partial_pairwise_align_selection_kl.csv"),
-                ("pca", "partial_pairwise_align_pca.csv"),
-                ("glmpca", "partial_pairwise_align_glmpca.csv"),
-            ],
-        )
-    if "armijo" in metafunc.fixturenames:
-        metafunc.parametrize(
-            "armijo, expected_log, filename",
-            [
-                (
-                    False,
-                    {
-                        "err": [0.047201842558232954],
-                        "loss": [
-                            52.31031712851437,
-                            35.35388862002473,
-                            30.84819243143108,
-                            30.770197475353303,
-                            30.7643461256797,
-                            30.76336403641352,
-                            30.76332791868975,
-                            30.762808654741757,
-                            30.762727812006336,
-                            30.762727812006336,
-                        ],
-                        "partial_fgw_cost": 30.762727812006336,
-                    },
-                    "partial_fused_gromov_wasserstein.csv",
-                ),
-                (
-                    True,
-                    {
-                        "err": [0.047201842558232954, 9.659795787581263e-08],
-                        "loss": [
-                            53.40351168112148,
-                            35.56234792074653,
-                            30.897730857089122,
-                            30.77217881677637,
-                            30.764588004718373,
-                            30.763380009717963,
-                            30.76332859918154,
-                            30.762818343959903,
-                            30.762728863994322,
-                            30.76272782254089,
-                            30.76272781211168,
-                        ],
-                        "partial_fgw_cost": 30.76272781211168,
-                    },
-                    "partial_fused_gromov_wasserstein_true.csv",
-                ),
-            ],
-        )
-
-
-def test_partial_pairwise_align(slices2, dissimilarity, filename):
+@patch("paste3.paste2.dissimilarity_metric")
```

Comment:
Since `dissimilarity_metric` is being tested elsewhere, its return value can be mocked here.